project/Build.scala

```diff
@@ -1440,6 +1459,73 @@ object Build {
       settings(commonBenchmarkSettings).
       enablePlugins(JmhPlugin)
 
+    def asScala3doc: Project = {
+      def generateDocumentation(targets: String, name: String, outDir: String, params: String = "") = Def.taskDyn {
+          val sourceMapping = "=https://github.com/lampepfl/dotty/tree/master#L"
+          run.in(Compile).toTask(s""" -o output/$outDir -t $targets -n "$name" -s $sourceMapping $params""")
+      }
+
+      project.settings(commonBootstrappedSettings).
+        dependsOn(`scala3-compiler-bootstrapped`).
+        dependsOn(`scala3-tasty-inspector`).
+        settings(
+          resolvers += Resolver.jcenterRepo,
+          resolvers += Resolver.bintrayRepo("kotlin", "kotlin-dev"),
+          resolvers += Resolver.bintrayRepo("virtuslab", "dokka"),
```

Comment:
Could you add a comment explaining what we need each of these resolvers for? Is this only temporary?

Reply:
sure